CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [1.11.0] - 2019-09-11
+### Added
+- Receiving: inconsistent events raises `TableSync::UnprovidedEventTargetKeysError`
+  (events that includes only a part of target keys (or nothing of them))
```

Comment:
"events that include only some of the target keys (or none)"

Reply:
cheta ia daun